turbopack/crates/turbopack-ecmascript/tests/analyzer/graph/iife-2/input.js

```diff
@@ -0,0 +1,15 @@
+var root = {};
```

Comment:
Can you make this test case smaller? The snapshots can easily change and are hard to check for correctness if there are so many statements